.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+For bug reports, please respect the following guidelines and check the boxes accordingly.
+
+For questions of the type: "How can this be done?", please consider posting them on Stackoverflow.
```

Comment:
I don't know SO so well, but can we make a link to the relevant "tag" or whatever they call it, or possibly tell people to tag their relevant questions 'mlr'?